source/iostreams.tex

```diff
@@ -16277,8 +16278,8 @@
 
 \pnum
 \begin{note}
-A postcondition of \tcode{last_write_time(p) == new_time} is not specified since it might not hold for file systems
-  with coarse time granularity.
+A postcondition of \tcode{last_write_time(p) == new_time} is not specified
+since it is possible not to hold for file systems with coarse time granularity.
```

Comment:
Another "possible not to" phrase, which I find awkward.

Maybe: "No postcondition of `last_write_time(p) == new_time` is specified, because a file system with coarse time granularity cannot represent all values of `file_time_type` and can store a value that is less than `new_time`."

What is a "coarse time granularity" though? It would be conforming for `file_time_type` to use picosecond resolution, in which case the postcondition wouldn't be possible even file systems with nanosecond resolution, which is not "coarse" by most definitions.

Maybe just:

> _Note:_ Sets the greatest value supported by the file system that is not greater than `new_time`. Some values cannot be represented if the file system's timestamps have coarser granularity than `file_time_type`.

This isn't a normative change, because we already say it's done as if by [`futimens`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/futimens.html) which has the "greatest value <= `new_time`" requirement.

Reply:
less intrusive suggestion pushed:
"since it does not necessarily hold for file systems with coarse time granularity"